VMware.CloudFoundation.PowerManagement.psm1

```diff
@@ -744,6 +744,104 @@ Function Set-VsanClusterPowerStatus {
+                $requestID = $respond.content | ConvertFrom-JSON
+                Write-PowerManagementLogMessage -Type INFO -Message "VxRail Cluster shutdown request accepted(ID:$($requestID.request_id))"
+                $uri2 = "https://$server/rest/vxm/v1/requests/$($requestID.request_id)"
+                $loopCounter = 0
+                while ($loopCounter -lt 13){
+                    $respond2 = Invoke-WebRequest -Method GET -Uri $uri2 -Headers $headers -UseBasicParsing
+                    if ($respond2.StatusCode -eq "202" -or $respond2.StatusCode -eq "200") {
+                        $checkProgress = $respond2.content | ConvertFrom-JSON
+                        if($checkProgress.state -match "COMPLETED" -or $checkProgress.state -match "FAILED" ) {
+                            break
+                        }
+                    }
+                    sleep(10)
+                    $loopCounter += 1
+                }
+                
+                if($checkProgress.extension.passed -match "true") {
+                    Write-PowerManagementLogMessage -Type INFO -Message "VxRail Cluster shutdown dry run: SUCCESSED."
+                    Write-PowerManagementLogMessage -Type INFO -Message "Starting VxRail Cluster shutdown."
+
+                    $respond = Invoke-WebRequest -Method POST -Uri $uri -Headers $headers -Body $payloadRun -UseBasicParsing
+                    if ($respond.StatusCode -eq "202" -or $respond.StatusCode -eq "200") {
+                        return $true
+                    } else {
+                        Write-PowerManagementLogMessage -Type ERROR -Message "Invoke VxRail Cluster shutdown: FAILED" -Colour Red
+                    }
+                }
+                else {
+					$errorMsg = ""
+					$checkProgress = $respond2.content | ConvertFrom-JSON
+                    $parsingError = $checkProgress.extension.status
+					foreach($errorElement in $parsingError)
+                    {
+                        if($errorElement.checkResult -match "FAILED")
+                        {
+                            $errorMsg = $errorMsg + "Label: $($errorElement.label),($($errorElement.checkResult)) `nMessage: $($errorElement.message)`n"
+                        }
+                    }
+					Write-PowerManagementLogMessage -Type INFO -Message "VxRail Cluster shutdown dry run: FAILED `n $errorMsg" -Colour Red
+                }
+            } else {
+                Write-PowerManagementLogMessage -Type ERROR -Message "Invoke VxRail Cluster shutdown: FAILED" -Colour Red
+            }
+        }
+        else {
+            Write-PowerManagementLogMessage -Type ERROR -Message "Connection to '$server' has failed. Check your environment and try again" -Colour Red
+        }
+    }
+    Catch {
+        Debug-CatchWriterForPowerManagement -object $_
+    }
+    Finally {
+        Write-PowerManagementLogMessage -Type INFO -Message "Completed the call to the Invoke-VxrailClusterShutdown cmdlet." -Colour Yellow
```

Comment:
Why is and `INFO` set to `Yellow`?

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108

i was following format similar to other cmdlets.  will raise an issue to review all log entry color scheme